src/ISDevice.cpp

```diff
@@ -808,6 +808,15 @@ bool ISDevice::WaitForFlashSynced(uint32_t timeout)
     unsigned int startMs = now;
 
     static unsigned int lastRequest = startMs;
+
+    if (forceSync)
+    {
+        sysParams.flashCfgChecksum = 0;
+        flashCfgUpload.checksum = 0;
+        flashCfgUploadTimeMs = 0;
+        printf("Forcing flash config sync.\n");
```

Comment:
SDK should avoid printing.  We don't know when or where this will be called, or how often.